signalr-go-server/pkg/signalr/jsonhubprotocol.go

```diff
@@ -5,10 +5,13 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/go-kit/kit/log"
```

Comment:
Why this dependency?

Reply:
Because of [jsonhubprotocol.go line 133](https://github.com/philippseith/signalr-ports/blob/69d1ee69690a8bd1fa3d957f6e6ebd1e6260562d/signalr-go-server/pkg/signalr/jsonhubprotocol.go#L133)